src/pages/manager/OpenAccountPage.js

```diff
@@ -1,11 +1,36 @@
 const { expect } = require('@playwright/test');
 
-export class AddCustomerPage {
+export class OpenAccountPage {
   constructor(page) {
     this.page = page; 
+    this.currencyDropdownLocator = page.getByTestId('currency');
+    this.customerDropdownLocator = page.getByTestId('userSelect');
+    this.processButton = page.getByRole('button', { name: 'Process' });
   }
 
   async open() {
     await this.page.goto('/angularJs-protractor/BankingProject/#/manager/openAccount');
   }
-}
\ No newline at end of file
+
+  async reload() {
+    await this.page.reload();
+  }
+
+  async selectCurrencyFromDropdown(currency) {
+    await this.currencyDropdownLocator.selectOption(currency);
+  }
+
+  async selectCustomerFromDropdown(customer) {
+    await this.customerDropdownLocator.selectOption(customer);
+  }
+
+  async assertDropdownHasValue(value) {
+    const selectedValue = await this.currencyDropdownLocator.inputValue();
+
+    await expect(selectedValue).toBe(value);
```

Comment:
The method `assertDropdownHasValue` uses `inputValue()` to get the selected value from a dropdown. This might not work as expected because `inputValue()` is typically used for input elements, not dropdowns. Consider using `this.currencyDropdownLocator.evaluate(node => node.value)` or `this.currencyDropdownLocator.selectOption()` to verify the selected option.

Reply:
checked, everything works properly